portal/client/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
Can you use the IBM plex font? check the qTrack or container repo scss file for configuration of same

Reply:
i see that you are using it in `portal/client/src/scss/global.scss`, lets remove this file